### ISSUE TITLE
Provide easy ways for users to download and install eBooks etc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [x011/android-daisy-epub-reader#12]
```
Currently eBooks are either installed using a low-level command line
interface (adb push) or they can be downloaded using a web browser on the
device. We will want to make the process significantly easier, ideally to
the point where users can browser and download books from various sources
at least as easily as purchasing a book from major online 'book' sellers.

Potentially we can separate the design of the application into several
components, where others can develop and provide complementary features
e.g. from specialist eBook providers such as BookShare.net and the RNIB.
The core book reader could be made to run as a service on the phone. And we
could provide one or more User Interfaces that suit the needs of various
types of user e.g. with a visual UI for sighted users and a UI that is
tuned to the needs of users who cannot see.

So, in summary, the redesign may include:
   * Book search and download
   * Book 'Reader', possibly running as a service, similar to the music player
   * User-Interface(s) to navigate the book being read, etc.

This work will follow the current integration of the revised SMIL parser,
etc. (ongoing work in r69)

```

Original issue reported on code.google.com by `julianharty` on 7 Jan 2010 at 11:38
